client/src/client.rs

```diff
@@ -121,9 +119,7 @@ impl Client {
                     }))
                     .await
             }
-            AuraeSocket::IPv6 { ip, scope_id } => {
-                let mut socket =
-                    ip.parse::<SocketAddrV6>().expect("invalid ip address");
+            AuraeSocket::IPv6 { ip: mut socket, scope_id } => {
```

Comment:
so does this path not work any more?

Reply:
It should work, if you have a way to specify the IPv6. With this PR, path has been brought back as being able to be specified, but a SocketAddr has not (partly because I don't know what a `scope_id` is and how we would represent an ip + scope_id in toml).

Reply:
I'm not sure what PR needed the changes as I haven't had enough time lately to keep up with them all. It was definitely working, but I assume the change was needed for something.

According to [this](https://networkengineering.stackexchange.com/questions/46653/what-is-the-use-of-the-ipv6-scope-id) (and the RFC it links), `%` is the separator used, so I'll use that. The examples of the scope id aren't u32 though.

Reply:
https://manpages.ubuntu.com/manpages/trusty/man7/ipv6.7.html <-- u32